### PR TITLE
Alfalfa with custom e+9.6 w/ py3.8 build

### PR DESCRIPTION
**Pull request by TShapinsky**
I had to learn how to build E+ in order to debug #338 So I figured I'd just make a quick build of the version we're using right now that doesn't require us to roll back python versions.

### CODE REVIEW

**Comment by laurenklun** [comment]
@kbenne  can we get your okay with this before we merge?

**Comment by kbenne** [comment]
This seems ok to me.

**Review by nllong — APPROVED** [review]
looks good. worked as expected.

File: alfalfa_worker/Dockerfile [hunk 1]
```diff
@@ -57,14 +53,14 @@ ENV OPENSTUDIO_DOWNLOAD_FILENAME OpenStudio-3.3.0+ad235ff36e-Ubuntu-20.04.deb
 ENV OPENSTUDIO_DOWNLOAD_URL https://github.com/NREL/OpenStudio/releases/download/v3.3.0/OpenStudio-3.3.0+ad235ff36e-Ubuntu-20.04.deb
 
 ENV ENERGYPLUS_VERSION 9.6.0
-ENV ENERGYPLUS_TAG v9.6.0
-ENV ENERGYPLUS_SHA f420c06a69
+ENV ENERGYPLUS_TAG 9.6-3.8
+ENV ENERGYPLUS_SHA e78d88a246
 ENV ENERGYPLUS_DIR /usr/local/EnergyPlus-9-6-0
```
Comment (nllong) [inline alfalfa_worker/Dockerfile:58]: bam!

File: alfalfa_worker/Dockerfile [hunk 2]
```diff
@@ -158,12 +154,10 @@ COPY pyproject.toml $BUILD_DIR
 COPY poetry.lock $BUILD_DIR
 
 # Reinstall/upgrade setuptools because this is needed for symfit (used in E+ Python)
-RUN ln -sf /usr/bin/python3.7 /usr/bin/python \
-    && ln -sf /usr/bin/python3.7 /usr/bin/python3 \
-    && ln -sf /usr/bin/pip3 /usr/bin/pip \
-    && python -m pip install poetry==1.1.12 setuptools==62.1.0 \
-    && python -m poetry config virtualenvs.create false \
-    && python -m poetry install --no-dev
+RUN ln -sf /usr/bin/python3 /usr/bin/python \
```
Comment (nllong) [inline alfalfa_worker/Dockerfile:157]: so much cleaner.

**Review by nllong — APPROVED** [review]
🏆

**Pull request merged**